resources/views/tags.twig

```diff
@@ -1,8 +1,9 @@
+{{ asset_add("styles.css", "anomaly.field_type.multiple::css/select2.min.css") }}
```

Comment:
Line 1 is the select2 css that comes from the vendor, 
Line 2 is the styling trying to match the theming to your old one.